src/CollAction/ValidationAttributes/FileSizeAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.ComponentModel.DataAnnotations;
+
+namespace CollAction.ValidationAttributes
+{
+    public class FileSizeAttribute : ValidationAttribute
```

Comment:
This seems like a decent idea, but is unused. Don't check in classes you don't use.